theme/style.css

```diff
@@ -20,3 +20,50 @@ img {
 }
 
 /* style your grid sections here!  */
+
+.content>div:nth-child(3) {
```

Comment:
For legibility, can you please put spaces around your direct child selectors (`>`)?